lib/MixedModels/LMM.rb

```diff
@@ -194,17 +220,32 @@ def LMM.from_formula(formula:, data:, weights: nil, offset: 0.0, reml: true,
     vars = rhs.split %r{\s*[+|()*]\s*}
     vars.delete("")
     vars.uniq!
+
+    #Do you want to impose any constraints on variable names? Are they allowed
+    #to start with numbers or contain symbols like "}"?
```

Comment:
I guess, I should exclude everything that cannot be converted to a symbol. I will look into that